Prepare for v0.8.0 release

llucax opened the pull request:
Bump the version in the cookiecutter template and update the release notes.

llucax committed:
Prepare for v0.8.0 release

Bump the version in the cookiecutter template and update the release
notes.

Signed-off-by: Leandro Lucarella <luca-frequenz@llucax.com>

llucax commented:
This one is to merge last, and will probably need to fix some conflicts, so setting as draft. But the contents can be reviewed now.

llucax commented:
Rebased and set as ready for review.

llucax commented:
Sorry,  I forgot to add upgrading instructions, will update soon again.

llucax commented:
Done!

daniel-zullo-frequenz commented on RELEASE_NOTES.md:
```diff
@@ -2,15 +2,33 @@
+
+* Update the `frequenz-repo-config` dependencies in `pyproject.toml` to 0.8.0.
+
+* Run in the root of your project:
```
What about clarify explicitly that is the root directory?
```suggestion
* Run in the root directory of your project:
```

daniel-zullo-frequenz commented on RELEASE_NOTES.md:
````diff
@@ -2,15 +2,33 @@
+    EOF
+    ```
+
+* Optionally go to the project's settings and replace branch protection rules with the new rulesets. See the new [GitHub configuration guide](https://frequenz-floss.github.io/frequenz-repo-config-python/v0.8/user-guide/start-a-new-project/github-configuration/#branches) for more details.
````
```suggestion
* Optionally go to the GitHub project's settings and replace branch protection rules with the new rulesets. See the new [GitHub configuration guide](https://frequenz-floss.github.io/frequenz-repo-config-python/v0.8/user-guide/start-a-new-project/github-configuration/#branches) for more details.
```

daniel-zullo-frequenz reviewed (dismissed):
I have 2 optional suggestions, LGTM

daniel-zullo-frequenz commented:
Sorry, I removed it from the merging queue in case you'd like to address the optional suggestions

llucax commented:
Updated with the suggestions and also updated the tests (they will probably still fail because the version in the dependencies doesn't exist yet, this can be only fixed when we split this repo).
